CMakeLists.txt

```diff
@@ -138,6 +138,10 @@ include(cmake/hdf5_target.cmake)
 find_package(SpFFT 1.0.2 CONFIG REQUIRED)
 find_package(SPLA 1.4.0 CONFIG REQUIRED)
 
+if (USE_MEMORY_POOL)
```

Comment:
Sorry for the drive-by comment (as I'm sure this applies to other options as well): *please* namespace your CMake variables.

Reply:
That is a big TODO :)